build.gradle

```diff
@@ -6,12 +6,12 @@ buildscript {
     ext.lifecycle_version = '2.6.2'
 
     // build & version
-    ext.buildNumber = 279
-    ext.versionNumber = "0.24.1"
+    ext.buildNumber = 285
```

Comment:
Why this it isn't handled automatically by CI?

Reply:
@TruszczynskiA I inherited this from Alex. I also thought that it would be great to automize